src/ceph-volume/ceph_volume/util/encryption.py

```diff
@@ -18,8 +19,18 @@ def set_dmcrypt_no_workqueue(target_version: str = '2.3.4') -> None:
     """
     command = ["cryptsetup", "--version"]
     out, err, rc = process.call(command)
+    pattern: str = r'\b\d+(\.\d+)*\b'
+
+    if rc:
+        raise RuntimeError("Can't retrieve cryptsetup version.")
+
     try:
-        if version.parse(out[0]) >= version.parse(f'cryptsetup {target_version}'):
+        cryptsetup_version = re.match(pattern, out[0])
+
+        if cryptsetup_version is None:
+            raise RuntimeError('Error while checking cryptsetup version.')
```

Comment:
Print the version that was returned? `pattern`?

Reply:
`pattern` is the regex, if `cryptsetup_version` is `None` it means nothing matched, maybe simply print `out` and/or `err` ?

Reply:
Right my bad, yes I meant to print the string being parsed